apps/fabric-website/homepage.htm

```diff
@@ -185,7 +185,7 @@
     var scripts = [appPath + entryPointFilename + (isDev ? '.js' : '.min.js')];
 
     if (!window.React) {
-      // Only needed for Fabric 5 and 6 (7 bundles React)
+      // Only needed for Fluent UI 5 and 6 (7 bundles React)
```

Comment:
```suggestion
      // Only needed for Fabric 5 and 6 (Fabric/Fluent UI 7 bundles React)
```

Reply:
On this note, you should verify that switching to older fabric website versions still works after this goes in.

Reply:
yup, certainly.